tailwind.config.ts

```diff
@@ -19,6 +19,10 @@ const config = {
       },
     },
     extend: {
+      spacing: {
+        "17": "4.25rem",
+        "21": "5.25rem",
+      },
```

Comment:
Adding these values to the tailwind config to set the correct min height for the tier row. Items have m-0.5 (0.125rem) and either h-16 (4rem) or h-20 (5rem), so the tier row would need a min height of 4.25rem or 5.25rem.

I'd prefer to remove the margin on the items entirely and use flex gap, but it seems the dnd library doesn't support gap :(